.github/workflows/ci.yml

```diff
@@ -175,6 +176,12 @@ jobs:
           - typo3-version: "^12.4"
             php-version: "8.3"
             composer-dependencies: highest
+          - typo3-version: "^12.4"
```

Comment:
Don't we need those new jobs also within GitLab?

Reply:
Yes, on it.